Database-Features/No write operation allowed in snapshot mode during DQL.md

```diff
@@ -0,0 +1,56 @@
+
+# No write operation allowed in snapshot mode during DQL
+
+## Problem
+
+DQL(SELECT) on the system tables in snapshot mode fails with below error 
+```
+GlobalTransactionRollback msg: No write operation allowed in snapshot mode.
+```
+
+## Solution
+
+During the Exasol DB version upgrade , some of the system tables are altered as a result views depending on them becomes invalid.
+By design, the first time query to an invalid view tries to validate it (write operation).
+Snapshot execution mode is a special transaction mode that, in particular, minimizes the amount of transaction conflicts (see [**Snapshot mode**](https://docs.exasol.com/db/latest/database_concepts/snapshot_mode.htm "snapshot")  ). However, by definition snapshot execution mode is read-only, so such SELECTs after the system upgrade will fail with mentioned error.
+
+If the SELECT/DQL is via some application, It will make sense for the application  to run a SELECT (without snapshot execution mode) against objects depending on Exasol system views that application is using in
+snapshot execution mode after every DB update. If needed, one can do it only for a subset of invalid views, like 
+
+-- Filter on schema name and potentially on object name is to be refined on application side
+
+```sql
```

Comment:
As you know from another PR, one might also look in `EXA_SYS_COLUMS` to also find invalid system views.

One may think about incorporating this information or simply referring to that existing article.